src/main/java/com/hubspot/jinjava/tree/parse/TagToken.java

```diff
@@ -15,20 +15,22 @@
  **********************************************************************/
 package com.hubspot.jinjava.tree.parse;
 
-import static com.hubspot.jinjava.tree.parse.TokenScannerSymbols.TOKEN_TAG;
-
+import com.hubspot.jinjava.interpret.JinjavaInterpreter;
 import com.hubspot.jinjava.interpret.TemplateSyntaxException;
 import com.hubspot.jinjava.util.WhitespaceUtils;
 
 public class TagToken extends Token {
   private static final long serialVersionUID = -4927751270481832992L;
+  private final int TOKEN_TAG;
```

Comment:
```suggestion
  private final int tokenTag;
```
only static variables should be in upper case.